ledger/metrics/collection/graphite/default_conf/storage-aggregation.conf

```diff
@@ -0,0 +1,32 @@
+# Aggregation methods for whisper files. Entries are scanned in order,
+# and first match wins. This file is scanned for changes every 60 seconds
+#
+#  [name]
+#  pattern = <regex>
+#  xFilesFactor = <float between 0 and 1>
+#  aggregationMethod = <average|sum|last|max|min>
+#
+#  name: Arbitrary unique name for the rule
+#  pattern: Regex pattern to match against the metric name
+#  xFilesFactor: Ratio of valid data points required for aggregation to the next retention to occur
+#  aggregationMethod: function to apply to data points for aggregation
+#
+[min]
+pattern = \.lower$
+xFilesFactor = 0.1
+aggregationMethod = min
+
+[max]
+pattern = \.upper(_\d+)?$
+xFilesFactor = 0.1
+aggregationMethod = max
+
+[sum]
+pattern = \.sum$
+xFilesFactor = 0
+aggregationMethod = sum
+
```

Comment:
I think that's the expected behavior for counters; they're _supposed_ to be cumulative. If you want averages I think you might want to use a meter instead.

Reply:
I have been using a `com.codahale.metrics.Meter`. Internally, it has a variable `count` that is incremented on every call to `Meter.mark()`. The `GraphiteReporter` reports the value of `count` to Graphite.

In my test, I was reporting 6 values per minute of the metric `my-metric.count` to Graphite. Graphite was only storing one value per minute. Due to the rule I have now deleted, it was summing up the 6 values instead of taking the average.

Looking at the source code of `com.codahale.metrics.Counter` it seems to be the same story for counters.

Reply:
As discussed over Slack, will extend README.md to explain the config changes.